File: help/eng_gb/exercicios_da_ficha_4/Ficha04_005/Ficha04_005.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Number     : " ;
    $num = intval(isset($_POST["num"])? $_POST["num"] : isset($_GET["num"]) ? $_GET["num"] : 0);
    
    echo $num . "<br/>";// print input 
    if($num == 0 ){
        echo  "Null" ;
    }else{
        if($num > 0 ){
            echo  "Positive" ;
        }else{
            echo  "Negative" ;
        }
    }
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	domingo, 2022 set 18 23:27

?>

File: help/eng_gb/exercicios_da_ficha_4/Ficha04_005/Ficha04_005.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha04_005 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Number \t: " );
        int num = Integer.valueOf(keyb.nextLine());
        if(num == 0 ){
            System.out.print("Null" );
        }else{
            if(num > 0 ){
                System.out.print("Positive" );
            }else{
                System.out.print("Negative" );
            }
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:27
